open-metadata-resources/open-metadata-deployment/charts/vdc-open/templates/igcproxy-deployment.yaml

```diff
@@ -0,0 +1,41 @@
+# <!-- SPDX-License-Identifier: Apache-2.0 -->
+# Copyright Contributors to the Egeria project.
```

Comment:
is this Virtualizer related?

Reply:
as previous - probably got pulled in due to pom change, but this is way backlevel